composer/callbacks/checkpoint_saver.py

```diff
@@ -244,6 +246,7 @@ def __init__(
         overwrite: bool = False,
         num_checkpoints_to_keep: int = -1,
         weights_only: bool = False,
+        save_planner: Optional[Any] = None
```

Comment:
can you leave a comment on why this is any, and open a jira to fix once we deprecated torch 1.13?